NetKAN/TweakScaleRescaled-SafetyNet.netkan

```diff
@@ -0,0 +1,17 @@
+spec_version: v1.18
+identifier: TweakScaleRescaled-SafetyNet
```

Comment:
Manual installs would be handled better if the DLL was `TweakScaleRescaled-SafetyNet.dll` or `TweakScaleRescaled_SafetyNet.dll` (the DLL detection code will change underscore to a dash when matching with identifiers, I believe).

Reply:
Fair enough